src/content/docs/en/reference/errors/static-client-address-not-available.mdx

```diff
@@ -18,7 +18,7 @@ import DontEditWarning from '~/components/DontEditWarning.astro'
 ## What went wrong?
 The `Astro.clientAddress` property is only available when [Server-side rendering](/en/guides/server-side-rendering/) is enabled.
 
-To get the user's IP address in static mode, different APIs such as [Ipify](https://www.ipify.org/) can be used in a [Client-side script](/en/core-concepts/astro-components/#client-side-scripts) or it may be possible to get the user's IP using a serverless function hosted on your hosting provider.
+To get the user's IP address in static mode, different APIs such as [Ipify](https://www.ipify.org/) can be used in a [Client-side script](/en/guides/client-side-scripts/) or it may be possible to get the user's IP using a serverless function hosted on your hosting provider.
```

Comment:
Was this changed in the error.ts file in the main Astro repo as well? If not, we have to change there as well!

Reply:
@Yan-Thomas You're right, and the good news is, no harm in linking to the page now because that one totally exists! Making the PR there, too.